utilities/search_index.py

```diff
@@ -38,6 +45,7 @@ def show_stat(definition, value):
         output("{:>25}:{}".format(definition, value))
 
     first_res_time = 0
+    nro = 0
```

Comment:
Esto está mal (y además da un "Results 1" erróno. Lo mejor es soportar que no tenés results saliendo con un `if not results` y listo...

Reply:
+1